### PR TITLE
chore: remove duplicate words

### PR DESCRIPTION
**Pull request by hattizai**
remove duplicate words in comments to improve readability.

### CODE REVIEW

File: src/tools/rustfmt/src/config/mod.rs
Comment (ytmimi) [inline src/tools/rustfmt/src/config/mod.rs:1]: This is pretty minor, but the rustfmt team prefers if these kinds of changes be made directly to https://github.com/rust-lang/rustfmt. Could you revet these changes and open up a PR there.

Comment (hattizai) [reply src/tools/rustfmt/src/config/mod.rs:1]: thanks, reverted.

**Comment by ytmimi** [comment]
@hattizai instead of a new commit can you amend the history to remove the rustfmt change?

**Comment by hattizai** [comment]
Done

**Comment by saethlin** [comment]
It's always amazing to me how many of these mistakes people manage to dig up. I suppose we just keep checking them in! Thanks for the fixes.

@bors r+

**Pull request merged**